lib/classes/squad_members/squad_member.dart

```diff
@@ -0,0 +1,23 @@
+class SquadMember {
+  bool _isInfected = false;
+  bool _isMutant = false;
+  int _countDaysInfected = 0;
+
+  bool get isInfected => _isInfected;
+
+  int get countDaysInfected => _countDaysInfected;
```

Comment:
you have setter and getter, so you are incapsulate nothing. remove incapsulation.